kubernetes_asyncio/client/rest.py

```diff
@@ -137,13 +137,15 @@ async def request(self, method, url, query_params=None, headers=None,
 
         # For `POST`, `PUT`, `PATCH`, `OPTIONS`, `DELETE`
         if method in ['POST', 'PUT', 'PATCH', 'OPTIONS', 'DELETE']:
-            if re.search('json', headers['Content-Type'], re.IGNORECASE):
+            if (re.search('json', headers['Content-Type'], re.IGNORECASE) or
+                    headers['Content-Type'] == 'application/apply-patch+yaml'):
                 if headers['Content-Type'] == 'application/json-patch+json':
                     if not isinstance(body, list):
                         headers['Content-Type'] = 'application/strategic-merge-patch+json'
+                request_body = None
```

Comment:
As is autogenerated let's try to keep a changeset as small as possible. I think is not necessary.

Reply:
Done